presto-main/src/main/java/io/prestosql/execution/DynamicFilterConfig.java

```diff
@@ -37,12 +37,12 @@
     private boolean enableLargeDynamicFilters;
     private int serviceThreadCount = 2;
 
-    private int smallBroadcastMaxDistinctValuesPerDriver = 100;
-    private DataSize smallBroadcastMaxSizePerDriver = DataSize.of(10, KILOBYTE);
-    private int smallBroadcastRangeRowLimitPerDriver;
-    private int smallPartitionedMaxDistinctValuesPerDriver = 10;
+    private int smallBroadcastMaxDistinctValuesPerDriver = 200;
+    private DataSize smallBroadcastMaxSizePerDriver = DataSize.of(20, KILOBYTE);
+    private int smallBroadcastRangeRowLimitPerDriver = 5000;
+    private int smallPartitionedMaxDistinctValuesPerDriver = 20;
     private DataSize smallPartitionedMaxSizePerDriver = DataSize.of(10, KILOBYTE);
-    private int smallPartitionedRangeRowLimitPerDriver = 100;
+    private int smallPartitionedRangeRowLimitPerDriver = 500;
```

Comment:
let's disable min/max collection due to: https://github.com/prestosql/presto/issues/5368

Reply:
let's also disable min/max collection for large DFs for now.

Reply:
I've disabled for small and large DF in case of broadcast joins. I think for partitioned joins we might still be ok as each driver is collecting a disjoint range ?